java/code_to_gast/java_router.py

```diff
@@ -2,13 +2,14 @@
 import java.code_to_gast.java_expression as java_expression
 import java.code_to_gast.java_helpers as java_helpers
 import java.code_to_gast.java_assign as java_assign
+import java.code_to_gast.java_conditional as java_conditional
 
 def node_to_gast(node):
     #base cases
     if type(node) == javalang.tree.Literal:
         # node.value is stored as string in java AST
         if node.value.isnumeric():
-            return {"type": "num", "value": node.value}
+            return {"type": "num", "value": int(node.value)}
```

Comment:
Is the `int()` cast going to be a problem with decimals?

Reply:
We are currently only handling ints, and have a ticket to change that to handle both

Reply:
Gotcha. Can you add a comment like TODO(<username>) or better TODO(<taiga ticket #>)